tests/JSMinTest.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+use JSMin\JSMin;
+
+class JsMinTest extends TestCase
```

Comment:
This is already in the JSMin project I think. Anyway we only need to make sure files are combined correctly.